packages/pharos-site/src/pages/faqs.mdx

```diff
@@ -0,0 +1,91 @@
+import PageSection from '../../src/components/statics/PageSection.tsx';
+
+<PageSection isHeader title="Pharos FAQ" />
+
+<PageSection title="How can Pharos help my development?" subSectionLevel={2}>
+  <p>
+    Pharos contains visual and behavioral components that are consistent with our design decisions
+    out of the box. Pharos also seeks to overcome the challenges Bonsai faces around framework
+    interoperability, code bloat, and interdependent components.
+  </p>
```

Comment:
This bit isn't contextualized enough for an external audience and I don't think it makes sense to keep it here:

```suggestion
  <p>
    Pharos contains visual and behavioral components that are consistent with our design decisions
    out of the box.
  </p>
```